### ISSUE TITLE
MUSIQ metric needs to be added with a PyTorch module

### ISSUE THREAD

**Issue by ethanweber** [ethanweber/nerfiller#13]
I used a TensorFlow model for the MUSIQ metric which is difficult to install. We should find a better one with PyTorch and update this part of the code to work properly.

https://github.com/ethanweber/nerfiller/blob/e017c6e99d379b4f488183659d7864af1f390847/nerfiller/utils/metrics.py#L173-L185
